src/module/actor/dataModelClasses/OseDataModelCharacterAC.test.js

```diff
@@ -0,0 +1,272 @@
+import OseDataModelCharacterAC from "./OseDataModelCharacterAC";
```

Comment:
Are these unit tests meant to be ran in quench?

We should be able to run these tests through the terminal so we can get them as part of CI/CD.

Reply:
They're meant to be run in Quench, yep!

It would be cool to get these tests running in the terminal (they're just Mocha/Chai tests, after all), but I think we should make that a separate issue/changeset. It'd take some effort to mock out stuff like `Item.implement`, unless someone knows a good way to pull that out of Foundry and into our test setup.

These tests should definitely part of any PRs going forward, though. I'm not sure how we'd enforce that.